docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx

```diff
@@ -0,0 +1,146 @@
+import { Tabs, TabItem, Table, TableHead, TableRow, TableCell, TableBody, View } from '@aws-amplify/ui-react';
+import { ResponsiveTable, ResponsiveTableCell } from '@/components/ResponsiveTable';
+
+## Full API Reference
+
+### FaceLivenessDetectorProps
+
+Below is the full list of props that can be used with the `FaceLivenessDetector` component. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L4).
+
+<ResponsiveTable highlightOnHover>
```

Comment:
Semi-related to the above. Think you can just the [`ReactPropsTable`](https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/components/propsTable/ReactPropsTable.tsx) component to avoid hard coding

Reply:
Switched to using react props table! thanks!